lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp

```diff
@@ -254,39 +254,14 @@ class SmartCard
 };
 
 /** Reader provides card reader information, status and gives access to the smart card in it. */
-class Reader
+struct Reader
 {
-public:
-    enum class Status {
-        UNAWARE,
-        IGNORE,
-        CHANGED,
-        UNKNOWN,
-        UNAVAILABLE,
-        EMPTY,
-        PRESENT,
-        ATRMATCH,
-        EXCLUSIVE,
-        INUSE,
-        MUTE,
-        UNPOWERED,
-        _
-    };
-
-    Reader(ContextPtr context, string_t name, byte_vector cardAtr, flag_set<Status> status);
-
     SmartCard::ptr connectToCard() const { return std::make_unique<SmartCard>(ctx, name, cardAtr); }
 
-    bool isCardInserted() const { return status[Status::PRESENT]; }
-
-    std::string statusString() const;
-
+    ContextPtr ctx;
     const string_t name;
     const byte_vector cardAtr;
-    const flag_set<Status> status;
-
-private:
-    ContextPtr ctx;
+    const bool isCardInserted;
```

Comment:
The conventional name seems to be "card present":

```suggestion
    const bool isCardPresent;
```